.github/workflows/toolchain.yml

```diff
@@ -54,6 +54,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       RUST_BACKTRACE: "1"
+      GITHUB_ACTIONS: "1"
```

Comment:
We shouldn't need to set this, it's set automatically by github: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

We could even just look for "CI" instead.

Reply:
Thanks, seems working